src/membrane-keypad-element.ts

```diff
@@ -129,8 +160,9 @@ export class MembraneKeypadElement extends LitElement {
     }
   }
 
-  private up(key: string) {
+  private up(key: string, event: any) {
```

Comment:
You can either use `UIEvent` or `KeyboardEvent | MouseEvent | TouchEvent` (I personally prefer the first one)

Reply:
To use either option, I had to declare 'event.currentTarget as SVGElement' to avoid TS error: Property 'classList' does not exist on type 'EventTarget'. I updated with the first option - UIEvent.